backend/loi-service/src/main/java/org/egov/works/web/controllers/LetterOfIndentApiController.java

```diff
@@ -0,0 +1,96 @@
+package org.egov.works.web.controllers;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.annotations.ApiParam;
+import org.egov.works.service.LetterOfIndentService;
+import org.egov.works.util.ResponseInfo;
+import org.egov.works.web.models.LetterOfIndentRequest;
+import org.egov.works.web.models.LetterOfIndentResponse;
+import org.egov.works.web.models.ResponseHeader;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.List;
+
+@javax.annotation.Generated(value = "org.egov.codegen.SpringBootCodegen", date = "2022-08-04T15:05:28.525+05:30")
+
+@Controller
+@RequestMapping("/eGovTrial/Letter-Of-Indent-Service/1.0.0")

```

Comment:
Replace `eGovTrial/...` with the actual context path.

Reply:
done